pintc/src/asm_gen/tests.rs

```diff
@@ -179,10 +179,7 @@ fn select() {
                   Stack(Push(0))
                   Stack(Push(1))
                   Access(DecisionVarRange)
-                  Stack(Push(69))
```

Comment:
A reminder to change those tests to avoid selects from getting optimized away.

Reply:
🤦, fixed and didn't push